lib/realmapi.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
all of the whitespace changes are uncalled for. also use tabs for indentation (in all the other places)